Makefile

```diff
@@ -285,6 +295,10 @@ PKGS_riscv64=pkg/alpine pkg/ipxe pkg/mkconf pkg/mkimage-iso-efi pkg/grub     \
 	     pkg/debug pkg/dom0-ztools pkg/gpt-tools pkg/storage-init
 PKGS=$(PKGS_$(ZARCH))
 
+# these are the packages that, when built, also need to be loaded into docker
+# if you need a pkg to be loaded into docker, in addition to the lkt cache, add it here
+PKGS_DOCKER_LOAD=mkconf mkimage-iso-efi mkimage-raw-efi mkrootfs-ext4 mkrootfs-squash eve
+
```

Comment:
Is there any downside in loading all of the packages we build into docker (and the lkt cache)?

Reply:
Just space utilization. Which is why I worked har with Petr to figure out which ones we truly need, and create a list of only those. The first 5 are really small; the last (`eve`) is not. 

We don't have much of a choice:

* if you want to `docker run` something, it has to be in docker image cache
* if you want something to have multiple architectures and a multi-arch index and be readable in its blobs (all of which we need), then docker image cache doesn't support it and you need lkt cache

That having been said, docker engine is finally moving there. Docker engine 22.06.0-beta (some variant of it) includes support for storing images in docker using containerd image cache, which goes a long way.

My estimate is that it will be a year or so until the support has everything we need and is in GA. Once it is, we can look at getting rid of lkt cache.